test/integration/setup/shell.js

```diff
@@ -50,13 +50,20 @@ module.exports = {
 	runMochaTests: function(testsPaths, cb) {
 		var child = child_process.spawn(
 			'node_modules/.bin/_mocha',
-			['--timeout', (8 * 60 * 1000).toString(), '--exit'].concat(testsPaths),
+			[
+				'--timeout',
+				(8 * 60 * 1000).toString(),
+				'--exit',
+				'--require',
+				'./test/setup.js',
+			].concat(testsPaths),
```

Comment:
@MaciejBaj Should not we just read this file instead of configuring these options again. This way the test setup will always stay in sync with functional tests 

https://github.com/LiskHQ/lisk/blob/1.0.0/test/mocha.opts